adafruit_il0373.py

```diff
@@ -90,7 +138,12 @@ class IL0373(displayio.EPaperDisplay):
     """
 
     def __init__(self, bus, swap_rams=False, **kwargs):
-        start_sequence = bytearray(_START_SEQUENCE)
+        if kwargs.get("grayscale", False):
+            start_sequence = bytearray(len(_START_SEQUENCE) + len(_GRAYSCALE_LUT))
+            start_sequence[: len(_START_SEQUENCE)] = _START_SEQUENCE
```

Comment:
grayscale startup is different than tricolor startup, i think it makes a difference

https://github.com/adafruit/Adafruit_EPD/blob/master/src/panels/ThinkInk_290_Grayscale4_T5.h#L8

Reply:
I patch the sequence for grayscale below on line 167. The only thing I don't change is the DC voltage. Could you try adding `start_sequence[32] = 0x12 # VCM DC voltage to -1 V` to that if.